src/main/java/baseball/baseballV1/util/InputNumberValidator.java

```diff
@@ -0,0 +1,43 @@
+package baseball.baseballV1.util;
+
+import java.util.List;
+
+public class InputNumberValidator {
+    private final String validationTarget;
+
+    public InputNumberValidator(String inputNumbers) {
+        this.validationTarget = inputNumbers;
+        isNullOrEmpty();
+        hasThreeDigits();
+        hasWrongCharacters();
+        hasDuplicatedNumber();
+    }
+
+    private void isNullOrEmpty() {
+        if (validationTarget == null || validationTarget.length() == 0) {
```

Comment:
```suggestion
        private static final int EMPTY_LENGTH = 0;
        if (Objects.isNull(validationTarget) || Objects.equals(validationTarget.length(), EMPTY_LENGTH)) {
```

1. 매직넘버 사용을 지양하시면 더 좋을 것 같아요!
2. == null -> 자바에서 기본적으로 지원하는 문법으로 사용해보세요!

Reply:
Objects 에서 기본적으로 제공하는 API.. isNull, eqauls() 를 알고 있었음에도 적용을 하지 못한 것 같아요! 
리마인드 하겠습니다:)